src/main/java/org/springframework/social/bitbucket/api/impl/BitBucketTemplate.java

```diff
@@ -30,6 +27,8 @@
 
     private PrivilegeOperations privilegeOperations;
 
+    private UsersOperations usersOperations;
```

Comment:
Seems that all those `*Operations` could actually be final. That would require to inline `initSubApis()` but it is probably worth it. (Again, not directly related to your PR, feel free to ignore).